config-connector/solutions/networking/helm/autoneg/README.md

```diff
@@ -0,0 +1,194 @@
+# Autoneg
+
+==================================================
+
+## NAME
+
+  autoneg
+
+## SYNOPSIS
+
+Autoneg solution uses network endpoint groups to provision load balancing across multiple clusters.
+This solution uses `./cluster/templates/autoneg.yaml` from [gke-autoneg-controller](https://github.com/GoogleCloudPlatform/gke-autoneg-controller).
+For demonstration purposes it uses a docker image with a simple Node.js service (bulankou/node-hello-world) with a single endpoint that prints out a message.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository
+  
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the autoneg folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/networking/helm/autoneg
+      ```
+
+## REQUIREMENTS
+
+1. [Helm](../../../README.md#helm)
+1. GKE Cluster with Config Connector. This solution assumes that all resources are installed in the same project, where the cluster with Config Connector is installed, and that load balancing resources are installed on the same cluster where Config Connector is installed. If you would like to configure your resources in a different project, the easiest approach would be to give your Config Connector service account (`cnrm-system`) owner permissions on this target project.
+1. If your Config Connector version is earlier than [1.11.2](https://github.com/GoogleCloudPlatform/k8s-config-connector/releases) you need to apply [this workaround](https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/78#issuecomment-577285402) to `iampolicymembers.iam.cnrm.cloud.google.com` CRD.
```

Comment:
We don't have version 1.11.2, but instead 1.12.0.

Reply:
fixed.